src/EditorFeatures/CSharpTest/ConvertTupleToStruct/ConvertTupleToStructTests.cs

```diff
@@ -50,7 +74,7 @@ class Test
 {
     void Method()
     {
-        var t1 = new {|Rename:NewStruct|}(a: 1, b: 2);
```

Comment:
It looks like `{|Rename: |}` was doing extra validation that's now lost?

Reply:
Yes.  It's a known limitation.  @sharwell  is tracking an issue to bring this back at some point.  When we do, we'd update tehse tests.